src/fty_sendmail.cc

```diff
@@ -141,27 +141,29 @@ int main(int argc, char** argv)
             exit(EXIT_FAILURE);
         }
 
-        if (zconfig_get(config, "malamute/endpoint", nullptr)) {
+        char* aux = zconfig_get(config, "malamute/endpoint", nullptr);
+        if (aux) {
             zstr_free(&endpoint);
-            endpoint = strdup(zconfig_get(config, "malamute/endpoint", nullptr));
+            endpoint = strdup(aux);
```

Comment:
aren't you missing zstr_free(&aux);
?

Reply:
here, aux is a réf. owned by the zconfig_t object...